site/en/docs/extensions/mv3/faq/index.md

```diff
@@ -10,45 +10,6 @@ If you don't find an answer to your question here, try the [Chrome Web Store FAQ
 [\[google-chrome-extension\] tag on Stack Overflow][2], the [chromium-extensions group][3], or the
 [store help][4].
 
-#### General {: #general }
```

Comment:
Same comment as above re: removing deep links

Reply:
And same reply :)